libs/portals/admin/ids-admin/src/module.tsx

```diff
@@ -0,0 +1,71 @@
+import { lazy } from 'react'
+
+import { PortalModule } from '@island.is/portals/core'
+import { IDSAdminPaths } from './lib/paths'
+import { AdminPortalScope } from '@island.is/auth/scopes'
+import Tenant from './screens/Tenant'
+import Applications from './components/Applications/Applications'
+import ApplicationsScreen from './screens/ApplicationsScreen'
+import { Features } from '@island.is/react/feature-flags'
+import { m } from './lib/messages'
+
+const IDSAdmin = lazy(() => import('./screens/IDSAdmin'))
+
+const allowedScopes: string[] = [AdminPortalScope.idsAdmin]
+
+export const idsAdminModule: PortalModule = {
+  name: m.idsAdmin,
+  layout: 'full',
+  featureFlag: Features.idsAdminNewPortal,
+  enabled({ userInfo }) {
+    return userInfo.scopes.some((scope) => allowedScopes.includes(scope))
+  },
+  routes() {
+    return [
+      {
+        name: m.idsAdmin,
+        path: IDSAdminPaths.IDSAdmin,
+        element: <IDSAdmin />,
+      },
+      {
+        name: m.applications,
```

Comment:
Should wait for more design before we scaffold the subnavigation and all of these subpages. Focus on launching the tenant page and work on some application page components.

Reply:
You are right, only added the Application because the cards where almost a copy paste of the tenant cards.